android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraViewManager.java

```diff
@@ -37,6 +37,11 @@ public void setType(RCTCameraView view, int type) {
         view.setCameraType(type);
     }
 
+    @ReactProp(name = "jpegQuality")
+    public void setJpegQuality(RCTCameraView view, int jpegQuality) {
+        view.setTorchMode(jpegQuality);
```

Comment:
Sorry I meant the `view.setTorchMode(jpegQuality)` appears to be a mistake. presumably it should be `view.setJpegQaulity`